reason_4.0/lib/core/classes/entity.php

```diff
@@ -211,18 +370,27 @@ function get_characteristics() // {{{
 	function get_value( $col, $refresh = true ) // {{{
 	{
 		if( empty( $this->_values ) )
+		{
 			$this->_values = get_entity_by_id( $this->_id );
+			$this->full_fetch_performed(true);
+		}
 		if( !empty( $this->_values[ $col ]) OR (isset($this->_values[$col]) AND strlen($this->_values[ $col ]) > 0) )
 			return $this->_values[ $col ];
 		elseif(!array_key_exists($col, $this->_values))
 		{
-			if ($refresh)
-			{			
-				return $this->get_value_refresh ($col);
+			// This logic aims to prevent a full fetch from being
+			// performed every time a delegated value is requested
+			if ($refresh && !$this->full_fetch_performed() )
```

Comment:
Am I following: this means that you would need to call `$this->full_fetch_performed(false)` for force a reload of the entity before calling `$this->get_value('something', true)`?